pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/GetOptionsForActivityInstanceQuestion.java

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.ddp.route;
```

Comment:
new route to query from DB REMOTE_AUTOCOMPLETE picklist options for give question and string